### PR TITLE
Update gems to support testing Puppet 8

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Comment by ekohl** [comment]
This doesn't run acceptance tests anymore.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -2,37 +2,30 @@ name: CI
...
 
 jobs:
   setup_matrix:
-    if: github.event_name != 'schedule' || github.repository_owner == 'theforeman'
-    name: 'Setup Test Matrix'
+    name: Static validations
     runs-on: ubuntu-latest
     outputs:
       beaker_setfiles: ${{ steps.get_outputs.outputs.beaker_setfiles }}
       puppet_major_versions: ${{ steps.get_outputs.outputs.puppet_major_versions }}
```
Comment (ekohl) [inline .github/workflows/ci.yml:12]: I think these were dropped in the latest puppet_metadata, so I need to replace this. Perhaps I need to solve the beaker facts properly instead.

**Comment by ekohl** [comment]
Now uses the reusable workflows again (replacing https://github.com/theforeman/puppet-pulpcore/pull/310). It probably still has a curl issue: https://github.com/theforeman/puppet-pulpcore/pull/310#issuecomment-1757451437

**Comment by ekohl** [comment]
I'll need to update the required check before merging this.

**Review by evgeni — APPROVED** [review]

**Pull request merged**